messages/en.json

```diff
@@ -132,5 +133,30 @@
       "returnByDescription": "Select how long you would like to borrow the item for.",
       "submit": "Submit"
     }
+  },
+  "shiftSchedule": {
+    "title": "Shift Schedule",
+    "administratorMenu": {
+      "administratorMenu": "Administrator Menu",
+      "clearShiftSchedule": "Clear shift schedule"
+    },
+    "scheduleTable": {
+      "time": "Time",
+      "monday": "Monday",
```

Comment:
Try moving this into a single key value pair using cardinal pluralization like we talked about. It can look something like this:
"day": "{day, plural, =0 {Monday} =1 {Tuesday} =1 {Wednesday} =1 {Thursday} other {Friday}}"